common/source/docs/common-geotagging-images-with-mission-planner.rst

```diff
@@ -4,312 +4,337 @@
+This is where the user can define the Field of View (FOV) and
+orientation information. This information is incorporated into the
+result files but it is not used or stored in the picture EXIF data.
 
-::
+The **AMSL Alt** checkbox allows to select either AGL Altitude (Above
+Ground Level) or AMSL Altitude (Above Mean Sea Level).
 
-    TimeMS = 2
-    Week = 3
-    Lat = 6
-    ...
+Action buttons
+--------------
 
-FOV and altitude reference
---------------------------
+-  **Pre-process** - When this button is clicked, Geotag tool will try
+   to extract and calculate pictures positions
+-  **Location Kml** - When this button is clicked, Geotag tool will open
+   Google Earth and show pictures position on a map so the user can
+   check their correct positioning.
+-  **GeoTag Images** - If everything was correct in previous steps and
+   pictures are well positioned in the map, this button will write the
+   position information in the EXIF data of every picture
 
-This is where the user can define the Field of View (FOV) and
-orientation information. This information is incorporated into the
-result files but it is not used or stored in the picture EXIF data.
+Process messages
+----------------
 
-The **AMSL Alt** checkbox allows to select either AGL Altitude (Above
-Ground Level) or AMSL Altitude (Above Mean Sea Level).
+This is the place where the tool will output information
 
-Action buttons
---------------
+Additional information
+======================
 
--  **Pre-process** - When this button is clicked, Geotag tool will try
-   to extract and calculate pictures positions
--  **Location Kml** - When this button is clicked, Geotag tool will open
-   Google Earth and show pictures position on a map so the user can
-   check their correct positioning.
--  **GeoTag Images** - If everything was correct in previous steps and
-   pictures are well positioned in the map, this button will write the
-   position information in the EXIF data of every picture
+This section contains links to additional information/information for
+working with specific cameras:
 
-Process messages
-----------------
+-  `Geotag GoPro Images with a Pixhawk Log File <http://tuffwing.com/support/geotag_gopro_images_with_pixhawk_log.html>`__
+   (tuffwing.com)
 
-This is the place where the tool will output information
+Alternative Commercial Hardware Solutions
+=========================================
 
-Additional information
-======================
+DROTAG - Airborne Projects
+--------------------------
 
-This section contains links to additional information/information for
-working with specific cameras:
+The `DROTAG <https://www.airborneprojects.com/product/drotag-onboard-image-tagging-computer/>`_ (airborneprojects.com) is a small board (55x27mm at 21g) that connects to an APM compatible Flight Controller (like Pixhawk) through the one of the any available TELEMETRY port and through a MicroUSB cable to the camera. That's it. Nothing more is needed. 
```

Comment:
This says that it is a small board, and that nothing more is needed ... but it doesn't say "needed for what"? Best to start by saying DROTAG is a board that you can add which will automatically geotag files when this is specified in mission planner (or similar)